src/Services/FeedbackService.php

```diff
@@ -644,4 +658,28 @@ private function getUserIdFromOrder($order)
 
         return 0;
     }
+
+    /**
+     * @param int $feedbackId
+     *
+     * @return bool
+     */
+    private function feedbackExists(int $feedbackId): bool
+    {
+        try {
+            /** @var Feedback $feedback */
+            $feedback = $this->feedbackRepository->getFeedback($feedbackId);
+        } catch (ModelNotFoundException) {}
```

Comment:
Not assigning a variable is valid for PHP 8 but to be safe, assign one

Reply:
Done.